config/default_config.yml

```diff
@@ -47,6 +47,15 @@ BANS:
   # Same as above but the other way around, remove a player from the blacklist will unban him
   unblacklist_does_unban: yes
 
+VAC_GAME_BANS:
+  steam_api_key: ""
```

Comment:
Steam API key config key value should not be under the VAC game ban config. It should either be entirely separate or stay inside of the .env